assets/js/modules/adsense/components/setup/AdBlockingRecoveryApp/SetupMain.test.js

```diff
@@ -186,6 +186,44 @@ describe( 'AdBlockingRecoverySetupCTAWidget - SetupMain', () => {
 				dashboardURL
 			);
 		} );
+
+		it( 'should return to AdSense settings page when `Cancel` button is clicked', async () => {
```

Comment:
This (`should return to AdSense settings page when `Cancel` button is clicked`) and the above test "`should return to dashboard when `Cancel` button is clicked`" sounds the same, we should differentiate between them by adding the perquisite to returning to the specific page in the test name.

Reply:
Updated ✅